src/logger.ts

```diff
@@ -1,8 +1,56 @@
 /**
- * A method that returns the default logger for this application.
- * Uses a function instead of a constant to allow mocking.
- * @returns The default logger for this application
+ * The default logger for this application.
+ * Can be set to any object that has `debug`, `info`, `warn`, and `error` methods,
+ * as long as it can be cast to a `Console`.
+ * @public
  */
-export function getLogger(): Console {
-	return console;
+const logger: Console = console;
+export default logger;
+
+/**
+ * A wrapper method for the `debug` endpoint of the default logger.
+ * In most situations, it wouldn't make sense to import a single logger endpoint,
+ * but separate exports are necessary for creating jest mocks for testing.
```

Comment:
I wouldn't say it's necessary, only that they make the test code nicer, only having to import the function directly rather than importing the logger, cast methods of that logger to `jest.Mock`, and using that in the tests.

For conciseness' sake, I think these exports are a reasonable middle ground. Good job!

Reply:
That's fine at runtime if `logger.debug` is a `jest.Mock` object, because that value doesn't need access to `logger` using a bound `this`. TS doesn't know at compile time that it's importing a mock, hence the need to cast to `jest.Mock`. Verbose, but it gets the job done in cases where importing the function directly isn't doable.